packages/phrase/src/phrase-api.ts

```diff
@@ -141,6 +145,10 @@ export async function pushTranslations(
     formData.append('update_translations', 'true');
     formData.append('update_descriptions', 'true');
 
+    if (autoTranslate) {
+      formData.append('autoTranslate', 'true');
```

Comment:
This field is all lowercase in the docs. https://developers.phrase.com/en/api/strings/uploads/upload-a-new-file#body-autotranslate
```suggestion
      formData.append('autotranslate', 'true');
```